action.yaml

```diff
@@ -11,6 +11,12 @@ inputs:
       are not supported, because partial cache restoration leads to a "snowball"
       effect.
     required: true
+  included-images:
+    description: >
+      If "all", all images will be included.
```

Comment:
I would change the default to `"new"` and clarify that we, to quote the README: "filter out Docker images that are present before the action is run, notably those [pre-cached by GitHub actions](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#cached-docker-images); only save Docker images pulled or built in the same job after the action is run." I don't see much reason to support `"all"` unless we get some use case for that from folks using self-hosted runners that aren't pre-caching Docker images.